### ISSUE TITLE
the epsilon in batchnorm is different in keras and original darknet

### ISSUE THREAD

**Issue by yangw1234** [qqwweee/keras-yolo3#451]
the default value of epsilon in keras batchnorm is 1e-3, while the value is 1e-5 in darknet. this may cause some value accuracy loss
